src/shared/utils/determineProgressColor.ts

```diff
@@ -6,17 +6,16 @@ export const determineProgressColor = ({
   lowerRange,
 }: {
   coverage: number | null
-  upperRange: number
-  lowerRange: number
+  upperRange?: number | null
```

Comment:
IMO a better way to deal with null inputs is to fall back to codecov's default indication range rather than just `return 'primary'`. The default indication range is lowerRange = 60, upperRange = 80. Lmk what you think.

Reply:
and coverage may be non-null based on what we decide in that other conversation